account_asset_management/models/date_range.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2009-2017 Noviat
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import time
+
+from odoo import api, models
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class DateRange(models.Model):
+    _inherit = 'date.range'
+
+    @api.model
+    def create(self, vals):
+        # TODO:
```

Comment:
TODO: explaing why this override is needed in a nice docstring :)

Reply:
cc @luc-demeyer

Reply:
So far I have seen 'TODO's in odoo code always as comments not docstrings. Is there a section in the OCA maintainer tools that states that we should now use docstrings to document TODO's ?

Reply:
is not about putting the TODO in the docstring, is about adding a new "TODO" for you, which is: explain/document what this override does :wink: 
Sorry if it was not really clear :sweat_drops: